apps/docs/src/guide/wallets/wallet-transferring.md

```diff
@@ -1,14 +1,57 @@
+## Transferring Assets Between Accounts
 
-Transferring assets between wallets is really simple within the SDK.
+The `transfer` method initiates a transaction request that transfers an asset from one wallet to another. This method requires three parameters:
+
+1. The receiver's wallet address
+2. The amount of the asset to be transferred
+3. The ID of the asset to be transferred (optional - defaults to the base asset ID)
+
+Upon execution, this function returns a promise that resolves to a transaction response. To wait for the transaction to be processed, call `response.wait()`.
+
+### Example
+
+Here is an example of how to use the `transfer` function:
+
+<<< @/../../docs-snippets/src/guide/cookbook/transferring-assets.test.ts#transferring-assets-1{ts:line-numbers}
+
+In the previous example, we used the `transfer` method which creates a `ScriptTransactionRequest`, populates its data with the provided transfer information and submits the transaction.
+
+However, there may be times when you need the Transaction ID before actually submitting it to the node. To achieve this, you can simply call the `createTransfer` method instead.
+
+This method also creates a `ScriptTransactionRequest` and populates it with the provided data but returns the request object prior to submission.
+
+<<< @/../../docs-snippets/src/guide/cookbook/transferring-assets.test.ts#transferring-assets-2{ts:line-numbers}
+
+> **Note**: Any changes made to a transaction request will alter the transaction ID. Therefore, you should only get the transaction ID after all modifications have been made.
+
+<<< @/../../docs-snippets/src/guide/cookbook/transferring-assets.test.ts#transferring-assets-3{ts:line-numbers}
+
+## Transferring Assets To Contracts
+
+When transferring assets to a deployed contract, we use the `transferToContract` method, which shares a similar parameter structure with the `transfer` method.
+
+However, instead of supplying the target wallet's address, as done in `destination.address` for the transfer method, we need to provide an instance of [Address](../types/address.md) created from the deployed contract id.
+
+If you have the [Contract](../contracts/) instance of the deployed contract, you can simply use its `id` property. However, if the contract was deployed with `forc deploy` or not by you, you will likely only have its ID in a hex string format. In such cases, you can create an [Address](../types/address.md) instance from the contract ID using `Address.fromAddressOrString('0x123...')`.
+
+Here's an example demonstrating how to use `transferToContract`:
+
+<<< @/../../docs-snippets/src/guide/cookbook/transferring-assets.test.ts#transferring-assets-4{ts:line-numbers}
+
+Always remember to call the `waitForResult()` function on the transaction response. That ensures the transaction has been mined successfully before proceeding.
+
+
+## Transferring Assets Between Wallets
+
+Transferring assets between wallets is straightforward within the SDK.
```

Comment:
We need to keep only one section for transferring assets between accounts/wallets. It is either:

- [Transferring Assets Between Accounts](https://github.com/FuelLabs/fuels-ts/blob/d3637f7ec248fcdbdf37ca2b358909b34a001a28/apps/docs/src/guide/wallets/wallet-transferring.md#transferring-assets-between-accounts)

Or

- [Transferring Assets Between Wallets](https://github.com/FuelLabs/fuels-ts/blob/d3637f7ec248fcdbdf37ca2b358909b34a001a28/apps/docs/src/guide/wallets/wallet-transferring.md#transferring-assets-between-wallets)

Reply:
Hey @mvares, let me know if this suggestion is not clear enough or if you have any further questions about it.